src/mpid/ch4/src/ch4r_callbacks.h

```diff
@@ -634,4 +634,43 @@ static inline int MPIDI_send_long_ack_target_msg_cb(int handler_id, void *am_hdr
     goto fn_exit;
 }
 
+#undef FUNCNAME
+#define FUNCNAME MPIDI_comm_abort_origin_cb
+#undef FCNAME
+#define FCNAME MPL_QUOTE(FUNCNAME)
+static inline int MPIDI_comm_abort_origin_cb(MPIR_Request * sreq)
+{
+    int mpi_errno = MPI_SUCCESS;
+    MPIR_FUNC_VERBOSE_STATE_DECL(MPID_STATE_MPIDI_COMM_ABORT_ORIGIN_CB);
+    MPIR_FUNC_VERBOSE_ENTER(MPID_STATE_MPIDI_COMM_ABORT_ORIGIN_CB);
+    MPID_Request_complete(sreq);
+    MPIR_FUNC_VERBOSE_EXIT(MPID_STATE_MPIDI_COMM_ABORT_ORIGIN_CB);
+    return mpi_errno;
+}
+
+#undef FUNCNAME
+#define FUNCNAME MPIDI_comm_abort_target_msg_cb
+#undef FCNAME
+#define FCNAME MPL_QUOTE(FUNCNAME)
+static inline int MPIDI_comm_abort_target_msg_cb(int handler_id, void *am_hdr,
+                                                 void **data,
+                                                 size_t * p_data_sz,
+                                                 int *is_contig,
+                                                 MPIDIG_am_target_cmpl_cb * target_cmpl_cb,
+                                                 MPIR_Request ** req)
+{
+    int mpi_errno = MPI_SUCCESS;
+    MPIDI_CH4U_hdr_t *hdr = (MPIDI_CH4U_hdr_t *) am_hdr;
+
+    MPIR_FUNC_VERBOSE_STATE_DECL(MPID_STATE_MPIDI_COMM_ABORT_TARGET_MSG_CB);
+    MPIR_FUNC_VERBOSE_ENTER(MPID_STATE_MPIDI_COMM_ABORT_TARGET_MSG_CB);
+
+  fn_exit:
```

Comment:
No need for `fn_exit, fn_fail` labels here. The compiler will just complain.

Reply:
Wouldn't the `MPIR_ERR_CHKANDSTMT` above need these labels? Or should we just remove all return code checkings from this function?

Reply:
I'm not seeing `MPIR_ERR_CHKANDSTMT` in this function.

Reply:
Oh, I was looking at the wrong function. Fixing it right away.

Reply:
Done. Also got rid of the unused `mpi_errno`.